webpack.config.js

```diff
@@ -324,10 +299,7 @@ function getWebpackConfig( {
 			new webpack.NormalModuleReplacementPlugin( /^path$/, 'path-browserify' ),
 			new webpack.IgnorePlugin( /^props$/ ),
 			isCalypsoClient && new webpack.IgnorePlugin( /^\.\/locale$/, /moment$/ ),
-			new MiniCssExtractPluginWithRTL( {
-				filename: cssFilename,
-				rtlEnabled: true,
-			} ),
+			SassConfig.plugin( { cssFilename } ),
 			new WebpackRTLPlugin( {
```

Comment:
we _may_ need to pull this into the stack as well? @jsnajdr how does this work again?

Reply:
This plugin goes through all CSS assets in the webpack compilation and runs them through RTLCSS, producing two versions: RTL and LTR. I.e., forking each asset into two. These are then minified.

`mini-css-extract-plugin-with-rtl` patches the part where the runtime code is generated: the runtime looks at the document (`<html dir="ltr">`) and loads the appropriate version of each CSS.

Both plugins should be a standard part of our build scripts. Every consumer needs them.

Reply:
Okay, sounds like we should move this plugin over to `packages/calypso-build/webpack/sass.js` then :+1:

Reply:
69f97b8fe9